lib/Optimizer/GraphOptimizer/Lower.cpp

```diff
@@ -1059,5 +1059,5 @@ void glow::lower(Function *F, CompilationContext &cctx, const Backend *B,
   }
 
   // Remove nodes that were lowered.
-  DCE().run(F);
+  FunctionPassManager("DCE_FPM", {getDCEPassConfig()}).run(F, cctx);
```

Comment:
Minor nit: Not a big deal, but may be introduce a `runDCEPass(CompliationContext &cctx)` helper function? It seems like you use this line of code in multiple places.